app/collins/models/AssetLog.scala

```diff
@@ -87,6 +87,10 @@ case class AssetLog(
 
   @Transient
   lazy val assetTag: String = asset.tag
+  // TODO(gabe): if this log is indexed (serialized first) before solr is
+  // updated with the asset document, this will throw! This can happen when
+  // creating a new asset then immediately performing some attribute sets
+  // which create logs.
```

Comment:
@evanelias this isn't new; it's been lurking since inception. Implications are that the tag set will fall (can't find asset to modify in solr index) until solr populates. Not cataclysmic, but definitely not a great consistency model to expose